update.sh

```diff
@@ -45,7 +45,8 @@ for version in "${versions[@]}"; do
 		echo >&2 "warning: cannot determine sha for $version (tried all of ${allVersions[*]}); skipping"
 		continue
 	fi
-	shaVal="$(echo "$shaPage" | sed -r "s/.*Ruby ${fullVersion}<\/a><br \/>\s*sha256: ([^<]+).*/\1/")"
+	versionReleasePage="$(curl -fsSL 'https://www.ruby-lang.org/en/downloads/releases/' | grep "<td>Ruby $fullVersion</td>" -A 2 | awk -F'"' '{print $2}' | xargs)"
```

Comment:
@yosifkit thanks for your review. Actually, saving the page into a variable will drop the "lines", with the correct newline char, I can handle much more easier, and we don't do this update everyday, I think the overhead is acceptable, especially switched to xz will save more traffic than the overhead. What do you think?